doc/api/util.md

```diff
@@ -1073,6 +1073,17 @@ The `TextEncoder` class is also available on the global object.
 UTF-8 encodes the `input` string and returns a `Uint8Array` containing the
 encoded bytes.
 
+### textEncoder.encodeInto([src], [dest])
+
+
+* `src` {string} The text to encode.
+* `dest` {Uint8Array} the array to hold the encode result
+* Returns: {TextEncoderEncodeIntoResult}
```

Comment:
I’d prefer to actually describe the object here, similar to [this](https://github.com/nodejs/node/blob/37d27486fce50bd82b6b5095af880d435ed308f8/doc/api/tls.md#tlssocketgetcipher)